### PR TITLE
Forces can be removed

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
Don't force it

### CODE REVIEW

**Review by labkey-susanh — APPROVED** [review]
Looks good as long as the omerointegration module is gone.

**Comment by labkey-adam** [comment]
Yep, I'll merge this after merging https://github.com/LabKey/DiscvrLabKeyModules/pull/116

**Pull request merged**
